test/fees/FeeRevertCases.t.sol

```diff
@@ -0,0 +1,288 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {Test} from 'forge-std/Test.sol';
+import {IERC20} from 'openzeppelin-contracts/contracts/token/ERC20/IERC20.sol';
+import {Router, IRouter} from 'src/Router.sol';
```

Comment:
```suggestion
import {Router} from 'src/Router.sol';
```

Reply:
`IRouter` does use in  `FeeRevertCases.t.sol`.